hmmlearn/base.py

```diff
@@ -102,6 +102,9 @@ def __init__(self, n_components=1, startprob=None, transmat=None,
         self.transmat_prior = transmat_prior
         self.algorithm = algorithm
         self.random_state = random_state
+        self.logprob_ = None
+        self.n_iterations_performed = None
```

Comment:
n_iterations_performed would not be len(self.logprob_). In self.logprob_ I am only saving the last value of the log prob (it is not an array).